File: 0169-majority-element/0169-majority-element.java
class Solution {
    public int majorityElement(int[] nums) {

        Arrays.sort(nums);
        return nums[nums.length/2];
        // HashMap<Integer,Integer> map=new HashMap<>();
        // for(int i:nums){
        //     map.put(i,map.getOrDefault(i,0)+1);
        // }
        // int max=0,x=-1;;
        // for(int i=0;i<nums.length;i++){
        //     if(map.get(nums[i])>max){
        //         max=map.get(nums[i]);
        //         x=nums[i];
        //     }
        // }
        // return x;
    }
}

File: 0169-majority-element/0169-majority-element.py
class Solution(object):
    def majorityElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        # nums.sort()
        # c=0
        # for i in range(len(nums)-2):
        #     if nums[i==nums[i+1]]:
        #         c+=c
        #     if c>len(nums)/2:
        #         return nums[i]
        #     else:
        #         c=1
        # a=nums[len(nums)/2]
        # return a
        nums.sort()
        return nums[len(nums)//2]
        